default/methods/configs/mailgun_is_set.py

```diff
@@ -0,0 +1,10 @@
+from methods import routes
+from shared.settings import settings
+
+@routes.route('/api/configs/is-mailer-set')
+def mailgun_is_set():
+    is_set = settings.MAILGUN_KEY and settings.EMAIL_DOMAIN_NAME and settings != 'localhost'
+    
+    if is_set:
+        return "true"
```

Comment:
I think here you could  use the `jsonify` method for flask, as we always assume that the server responses will be in a valid JSON format. so you could add something like `{"result": true}` just to follow the convetion on sending JSON as responses always.

Reply:
In this case I think you'll also have to modify a bit your frontend methods to receive the value in the new format.

Reply:
Of course, will do that